specifyweb/frontend/js_src/lib/components/Formatters/Formatter.tsx

```diff
@@ -178,13 +178,19 @@ function Definitions({
               (fields): void => handleChanged({ value, fields }, index),
             ]}
             table={table}
-            onDelete={
-              trimmedFields.length < 2
-                ? undefined
-                : (): void =>
-                    handleChange(removeItem(formatter.definition.fields, index))
-            }
           />
+          <div className="flex">
+            {index === 0 ? null : (
+              <Button.Danger
+                onClick={(): void =>
+                  handleChange(removeItem(formatter.definition.fields, index))
+                }
+              >
+                {resourcesText.deleteDefinition()}
+              </Button.Danger>
+            )}
+            <span className="-ml-2 flex-1" />
```

Comment:
is this line necessary?

oh, is that to prevent button from being full-width?
if so, just replace `<div className="flex">` with `<div className="inline-flex">`

Reply:
yes indeed it was for that reason